test/test_compute_service.py

```diff
@@ -249,95 +249,102 @@ def test_block_device_mapping_launch_config(self):
     def test_block_device_mapping_attachments(self):
         name = "CBInstBlkAttch-{0}-{1}".format(
             self.provider.name,
-            uuid.uuid4())
+            uuid.uuid4().hex[:6])
 
         # Comment out BDM tests because OpenStack is not stable enough yet
         if True:
             if True:
 
-                # test_vol = self.provider.block_store.volumes.create(
-                #    name,
-                #    1,
-                #    helpers.get_provider_test_data(self.provider,
-                #                                   "placement"))
-                # with helpers.cleanup_action(lambda: test_vol.delete()):
-                #    test_vol.wait_till_ready()
-                #    test_snap = test_vol.create_snapshot(name=name,
-                #                                         description=name)
-                #
-                #    def cleanup_snap(snap):
-                #        snap.delete()
-                #        snap.wait_for(
-                #            [SnapshotState.UNKNOWN],
-                #            terminal_states=[SnapshotState.ERROR])
-                #
-                #    with helpers.cleanup_action(lambda:
-                #                                cleanup_snap(test_snap)):
-                #         test_snap.wait_till_ready()
-
-                lc = self.provider.compute.instances.create_launch_config()
```

Comment:
I think this deleted code should be restored. Although it's commented out, we plan to restore it once OpenStack's BDM handling becomes more stable. Ideally, these tests should pass on the Azure provider when uncommented.